### ISSUE TITLE
Error: concordance for multiple words

### ISSUE THREAD

**Issue by xdwang0726** [nltk/nltk#2966]
Hi, 

When I tried to use concordance to find multiple words following exact this official tutorial (https://www.nltk.org/howto/concordance.html), I got the error message 
```
AttributeError: 'list' object has no attribute 'lower'
```

Here is the codes I am using:
```
from nltk.corpus import gutenberg
from nltk.text import Text
corpus = gutenberg.words('melville-moby_dick.txt')
text = Text(corpus)
text.concordance(["monstrous", "size"])
```

nltk version 
```
nltk.__version__
Out[41]: '3.4.4'
```

Thanks!

**Comment by Biciato** [comment]
@xdwang0726 which python version are you using ?

**Comment by tomaarsen** [comment]
Hey @xdwang0726!
The changes that you mention were implemented in #2570, which I believe was first added in NLTK version 3.6. The documentation (which is for version 3.7) includes this accordingly, but your version of 3.4.4 does not have this functionality.

You can either upgrade to a newer version (NLTK 3.6 onwards), or e.g. call `text.concordance` or `text.concordance_list` multiple times, once per word.

I'll close this, as I don't think there's a bug here. That said, you're always free to open an issue or comment if you disagree or have something to add.

**Comment by xdwang0726** [comment]
Thanks, it works when I upgraded to NLTK 3.7
